### ISSUE TITLE
Strapi 5: Initial export in admin not working

### ISSUE THREAD

**Issue by Pijuli** [pluginpal/strapi-plugin-config-sync#144]
## Bug report
After installing the config sync plugin, when running yarn build I get the following error

`Rollup failed to resolve import "@strapi/helper-plugin" from "/home/node/node_modules/strapi-plugin-config-sync/admin/src/index.js".  This is most likely unintended because it can break your application at runtime.`

### Screenshots
![image](https://github.com/user-attachments/assets/60531903-341a-4e77-8173-006dcdb27088)

### System
- Node.js version: 18.20.4
- YARN version: 1.22.19
- Strapi version: 5.0.0
- Plugin version: 1.2.6
- Database:  PostgreSQL
- Operating system: Ubuntu24.04/Docker

**Comment by boazpoolman** [comment]
Yeah, the 2.0.0 beta should be used for v5, but it still has some issues, mainly on the admin side.

The CLI should work as expected, so you can use that in the mean time :)

**Comment by Pijuli** [comment]
I have some mates not really cli friends. I'll keep track on this! We are just starting the project, so no need to rush it. We'll be back when is fixed.

Thank you for the work!

**Comment by BayBreezy** [comment]
I just wanted to stop by and say thank you for the plugin. 
I am testing out v5 of strapi. Its pretty nice so far, Just waiting on this plugin and the email designer to kick in so that I can start making production apps. 
I will try out the cli.. even tho i am not sure how I am gonna make that work with a docker deployment on Coolify 🤔

**Comment by BayBreezy** [comment]
Just providing an update.
The CLI works like a charm in v5. I use Husky to export the content before committing & then in the `config/plugins.ts` file, I just use this config

```ts
  "config-sync": {
    enabled: true,
    config: {
      minify: true,
      excludedConfig: [
        "core-store.plugin_users-permissions_advanced",
        "core-store.plugin_users-permissions_grant",
        "core-store.plugin_upload_metrics",
        "core-store.strapi_content_types_schema",
        "core-store.ee_information",
      ],
      // This works when the container gets deployed.
      importOnBootstrap: true,
    },
  },
```
Thanks again.

**Comment by boazpoolman** [comment]
This has been fixed and released with the [stable version 2.0.0](https://github.com/pluginpal/strapi-plugin-config-sync/releases/tag/2.0.0).
